app/org/sagebionetworks/bridge/models/accounts/ParticipantOptionsLookup.java

```diff
@@ -21,6 +24,12 @@ public ParticipantOptionsLookup(Map<String,String> options) {
         this.options = options;
     }
     
+    public DateTimeZone getTimeZone(ParticipantOption option) {
+        String value = options.get(option.name());
+        // This method is null-safe (returns null)
+        return DateUtils.parseZoneFromOffsetString(value);
+    }
```

Comment:
Sorry, I changed a path where we're parsing JSON to throw a BadRequestException... here in the lookup we're assuming the values have been written correctly to DDB and they should come back correctly... if they don't and it's an IllegalArgumentException, that is translated to a 500 response and that's what we want. We want to catch it if it's wrong when we're reading the values from the user.